src/server/common/components/button/template.njk

```diff
@@ -1,9 +1,10 @@
 {% from "loader/macro.njk" import appLoader %}
 
 <span class="app-button__container">
-  <button class="govuk-button app-button"
+  <button class="govuk-button app-button{% if params.classes %} {{ params.classes }}{% endif %}"
           {% if params.name %}name="{{ params.name }}"{% endif %}
     {% if params.value %}value="{{ params.value }}"{% endif %}
+    {% if params.type %}type="{{ params.type }}"{% endif %}
```

Comment:
as mentioned no need to add type here. Submit is the default html button type

Reply:
done